src/engine/specification.md

```diff
@@ -63,22 +63,22 @@ This structure contains the attributes (in addition to the other parameters cont
   - `VALID` - given payload is valid
   - `INVALID` - given payload is invalid
   - `SYNCING` - sync process is in progress
+2. `validAncestorHash`: `DATA|None`, 32 Bytes - block hash of the most recent *valid* ancestor of the payload
 
 #### Specification
 
-1. Client software **MUST** validate the payload according to the execution environment rule set with modifications to this rule set defined in the [Block Validity](https://eips.ethereum.org/EIPS/eip-3675#block-validity) section of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) and respond with the validation result.
+1. Client software **MUST** validate the payload according to the execution environment rule set with modifications to this rule set defined in the [Block Validity](https://eips.ethereum.org/EIPS/eip-3675#block-validity) section of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) and return the validation result.
+    * If validation succeeds, return `{status: VALID, validAncestorHash: payload.blockHash}`
+    * If validation fails, return `{status: INVALID, validAncestorHash: ancestorHash}` where `ancestorHash` is the block hash of the most recent *valid* ancestor of the invalid payload. That is, the ancestor of the payload in the valid block tree with the highest `blockNumber`.
 
 2. Client software **MUST** discard the payload if it's deemed invalid.
 
-3. The call **MUST** return `SYNCING` status while the sync process is in progress and thus the execution cannot yet be validated.
-
-4. In the case when the parent block is unknown, client software **MUST** pull the block from the network and take one of the following actions depending on the parent block properties:
-  - If the parent block is a PoW block as per [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) definition, then all missing dependencies of the payload **MUST** be pulled from the network and validated accordingly. The call **MUST** be responded according to the validity of the payload and the chain of its ancestors.
-  - If the parent block is a PoS block as per [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) definition, then the call **MAY** be responded with `SYNCING` status and sync process **SHOULD** be initiated accordingly.
+3. Client software **MUST** return `{status: SYNCING, validAncestorHash: None}` if the client software does not have the requisite data available locally to validate the payload in less than `SLOTS_PER_SECOND / 30` (0.4s in the Mainnet configuration) or if the sync process is already in progress. In the event that requisite data to validate the payload is missing (e.g. does not have payload identified by `parentHash`), the client software **SHOULD** initiate the sync process.
```

Comment:
It might read as if a client MUST execute a payload in `SLOTS_PER_SECOND / 30` period of time, otherwise, return SYNCING. We might want to add more clarity around this parameter, like as follows: "if the requisite data are unavailable locally and the client software can't pull them from the network in `SLOTS_PER_SECOND / 30` period"